homework-submission/react-hw4/src/components/stores.js

```diff
@@ -0,0 +1,39 @@
+import { observable, action, computed, configure } from "mobx";
+import Todos from "./todos.json";
+
+configure({ enforceActions: true });
+
+class Stores {
+  @observable
+  todos = Todos;
+
+  @action
+  checker = item => {
+    item.done = !item.done;
+  };
+
+  @action
+  remover = item => {
+    const remove = this.todos.filter(elem => elem !== item);
```

Comment:
Give meaningful names to your variables/functions/classes!, `const remove` can be `const todosList ...`

Reply:
You should not compare `elem, item` because they are full objects .. so in this case you are comparing something like this `{} === {}` which is **always false**.. you should compare using primitive values within these objects, like `elem.id === item.id`.